index.js

```diff
@@ -31,6 +31,10 @@ function waitConnection (socket, write) {
   }
 }
 
+function isExternalUrl (url = '') {
+  return urlPattern.test(url)
```

Comment:
How is this guaranteed to be "an external url"? Shouldn't the headers be inspected to determine if there is a `3xx` code that should be honored?

Reply:
That was my initial thought but `status code` is not getting passed through the `rewriteHeaders` method

Reply:
Then I would think the solution starts with resolving that issue. I do not see how this simple test is a guarantee of a redirect.

Reply:
If `status code` is necessary for the checking, I think you can start a PR in [`fastify-reply-form`](https://github.com/fastify/fastify-reply-from) to achieve it.